examples/acephei/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "federation-demo",
+  "description": "Full federation demo to illuminate good design patterns and practices.",
+  "scripts": {
+    "watch": "tsc --build tsconfig.json --watch",
+    "start": "nodemon -e js,graphql,json --watch services start.js",
+    "postinstall": "tsc --build tsconfig.json",
+    "check": "concurrently \"npm:check:*\"",
+    "push": "concurrently \"npm:push:*\"",
+    "clean": "git clean -dfqX -- ./node_modules **/{dist,node_modules}/ ./{services,clients}/*/tsconfig.tsbuildinfo ./__tmp__*",
+    "check:accounts": "npx apollo service:check --localSchemaFile=./services/accounts/schema.graphql --serviceName=accounts",
+    "check:books": "npx apollo service:check --localSchemaFile=./services/books/schema.graphql --serviceName=books",
+    "check:products": "npx apollo service:check --localSchemaFile=./services/products/schema.graphql --serviceName=products",
+    "check:reviews": "npx apollo service:check --localSchemaFile=./services/reviews/schema.graphql --serviceName=reviews",
+    "push:accounts": "npx apollo service:push --localSchemaFile=./services/accounts/schema.graphql --serviceName=accounts --serviceURL=https://acephei-accounts.herokuapp.com",
+    "push:books": "npx apollo service:push --localSchemaFile=./services/books/schema.graphql --serviceName=books --serviceURL=https://acephei-books.herokuapp.com",
+    "push:products": "npx apollo service:push --localSchemaFile=./services/products/schema.graphql --serviceName=products --serviceURL=https://acephei-products.herokuapp.com",
+    "push:reviews": "npx apollo service:push --localSchemaFile=./services/reviews/schema.graphql --serviceName=reviews --serviceURL=https://acephei-reviews.herokuapp.com"
+  },
+  "dependencies": {
+    "accounts": "file:services/accounts",
+    "books": "file:services/books",
+    "gateway": "file:services/gateway",
+    "products": "file:services/products",
+    "reviews": "file:services/reviews"
+  },
+  "devDependencies": {
+    "@types/react": "^16.9.9",
+    "concurrently": "^4.1.2",
```

Comment:
I agree with this one, I might need some help with some issues I was seeing from duplicate `graphql` modules. The `apollo` tooling `devDependency` was giving duplicate module errors when trying to run the acephei gateway.

Reply:
Since we aren't going to be running this in CI, I think it's okay to keep the `npx` for now. @abernix this is the last comment of feedback. Let me know if you think this is okay and I'll resolve this comment.

Reply:
Oh, right.  Thank you.  I forgot about the two-copies of `graphql` conflict, even though it has been a low-key bane of its (it being `apollo`, the CLIs) existence and a pressing reason why it needs to be `npx`'d.